IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -60,14 +60,12 @@
                         <td>@brr.Longitude</td>
                         <td>@brr.Latitude</td>
                         <td>@brr.Phone</td>
-                        <td>@brr.WebsiteUrl</td>
-                        <td>@brr.UpdatedAt</td>
-                        <td>@brr.CreatedAt</td>
-
+                        <td><a href="@brr.WebsiteUrl">@brr.WebsiteUrl</a></td>

```

Comment:
I added anchor tag <a> to create a hyperlink to link to the website of the corresponding brewery. I made this change from user point of view. This will help the users to navigate directly to the website of a brewery they like for more details.
https://www.w3schools.com/tags/tag_a.asp